.codeclimate.yml

```diff
@@ -0,0 +1,20 @@
+#
+# ---Choose Your Languages---
+# To disable analysis for a certain language, set the language to `false`.
+# For help setting your languages:
+# http://docs.codeclimate.com/article/169-configuring-analysis-languages
+#
+languages:
+   Ruby: false
+   JavaScript: true
+   Python: false
+   PHP: false
+#
+# ---Exclude Files or Directories---
+# List the files or directories you would like excluded from our analysis.
+# For help setting your exclude paths:
+# http://docs.codeclimate.com/article/166-excluding-files-folders
+#
+exclude_paths:
+ - 'build/*'
```

Comment:
Code Climate will only work after it is merged to master. At least from what I can see.